content/self-host/rustdesk-server-pro/smtp/Microsoft365/_index.en.md

```
@@ -22,7 +22,7 @@ For general SMTP setup, see [SMTP](../).
 | OAuth2 Client secret | The secret `Value` created under `Certificates & secrets` |
 
 This screenshot shows where these values are entered in RustDesk:
-![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft 365/images/21-RustDesk-SMTP-OAuth2-2.png)
+![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft365/images/21-RustDesk-SMTP-OAuth2-2.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)